rfhub2/cli/keywords/keywords_importer.py

```diff
@@ -48,8 +48,7 @@ def _delete_collections(self) -> Set[int]:
         """
         collections = self.client.get_collections()
         collections_id = {collection["id"] for collection in collections}
-        for id in collections_id:
-            self.client.delete_collection(id)
+        self.client.delete_all_collection()
```

Comment:
I think that this method might not be needed anymore at all. All it was doing is done by `delete_all_collections` method

Reply:
true